taxi/views.py

```diff
@@ -87,3 +89,28 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    form_class = DriverCreationForm
+
+
+class DriverDeleteView(LoginRequiredMixin, generic.DeleteView):
+    model = Driver
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+class DriverLicenseUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Driver
+    form_class = DriverLicenseUpdateForm
+
+
+def car_assign_driver(request: HttpRequest, pk: int) -> HttpResponse:
+    Car.objects.get(id=pk).drivers.add(request.user)
+    return redirect(f"/cars/{pk}/")
+
+
+def car_delete_driver(request: HttpRequest, pk: int) -> HttpResponse:
+    Car.objects.get(id=pk).drivers.remove(request.user)
+    return redirect(f"/cars/{pk}/")
```

Comment:
You can make it as one function and change template accordingly.
And more, try not to use hardcoded string like this, use redirect to predefined `app:location` from your urls.